app/Http/Livewire/AllProjects.php

```diff
@@ -0,0 +1,236 @@
+            }
+        );
+    }
+
+    public function updatedSeekings()
+    {
+        if (! is_array($this->seekings)) {
+            return;
+        }
+        $this->seekings = array_filter($this->seekings,
+            function ($seeking) {
+                return $seeking != false;
+            }
+        );
+    }
+
+    public function updatedInitiators()
+    {
+        if (! is_array($this->initiators)) {
+            return;
+        }
+        $this->initiators = array_filter($this->initiators,
+            function ($initiator) {
+                return $initiator != false;
+            }
+        );
+    }
+
+    public function updatedSeekingGroups()
+    {
+        if (! is_array($this->seekingGroups)) {
+            return;
+        }
+        $this->seekingGroups = array_filter($this->seekingGroups,
+            function ($seekingGroup) {
+                return $seekingGroup != false;
+            }
+        );
+    }
+
+    public function updatedMeetingTypes()
+    {
+        if (! is_array($this->meetingTypes)) {
+            return;
+        }
+        $this->meetingTypes = array_filter($this->meetingTypes,
+            function ($meetingType) {
+                return $meetingType != false;
+            }
+        );
+    }
+
+    public function updatedLocations()
+    {
+        if (! is_array($this->locations)) {
+            return;
+        }
+        $this->locations = array_filter($this->locations,
+            function ($location) {
+                return $location != false;
+            }
+        );
+    }
+
+    public function updatedCompensations()
+    {
+        if (! is_array($this->compensations)) {
+            return;
+        }
+        $this->compensations = array_filter($this->compensations,
+            function ($compensation) {
+                return $compensation != false;
+            }
+        );
+    }
+
+    public function updatedSectors()
+    {
+        if (! is_array($this->sectors)) {
+            return;
+        }
+        $this->sectors = array_filter($this->sectors,
+            function ($sector) {
+                return $sector != false;
+            }
+        );
+    }
+
+    public function updatedImpacts()
+    {
+        if (! is_array($this->impacts)) {
+            return;
+        }
+        $this->impacts = array_filter($this->impacts,
+            function ($impact) {
+                return $impact != false;
+            }
+        );
+    }
+
+    public function updatedRecruitmentMethods()
+    {
+        if (! is_array($this->recruitmentMethods)) {
+            return;
+        }
+        $this->recruitmentMethods = array_filter($this->recruitmentMethods,
+            function ($recruitmentMethod) {
+                return $recruitmentMethod != false;
+            }
+        );
+    }
+
+    public function render()
+    {
+        return view('livewire.all-projects', [
+            'projects' => Project::status('published')
+                    ->where(
```

Comment:
I think this should only be applied with a `when` clause only when the search query is present, otherwise there's an extra database query just to search for items which match an empty string.